test/files/sample.ts

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
I use it in check-bin.sh for the new tests - I was thinking we should have a file for that instead of linting configuration.ts which seems somewhat arbitrary